src/fhir/testUtilities.tsx

```diff
@@ -81,8 +90,8 @@ export const deleteFhirData = async (resource?: string, id?: string) => {
   for (const r of resources) {
     const fhirData = await getResources(r as string);
     if (!("entry" in fhirData)) {
-      console.debug("Nothing to delete; no data in database");
-      return;
+      console.debug(`No ${r} to delete`);
```

Comment:
is it continue since it loops through different resources now?

Reply:
yeah otherwise if there's no patient, then it won't delete any practitioners. So we want to continue iterating through the resource types